components/Layout.js

```diff
@@ -1,33 +1,38 @@
 import styled from "styled-components";
 import Header from "./Header";
-import Main from "./Main";
 import NavigationBar from "./NavigationBar";
+import WallShred from "./Wallshred";
+
 export default function Layout({ children }) {
   return (
-    <StyledLayout>
-      <Header />
-      <NavigationLayout></NavigationLayout>
-      <MainLayout>{children}</MainLayout>
-    </StyledLayout>
+    <>
+      <WallShred />
+      <StyledLayout>
+        <Header />
+        <NavigationBar></NavigationBar>
```

Comment:
If ```NavigationBar```doesn't have children, you can write it with a self closing tag

Reply:
yeah that sprung out alot automatically and i dont care to change it anymore to self closing tag 😆